Client/Application/AppDelegate.swift

```diff
@@ -62,28 +62,37 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     private func setUserAgent() {
         let webView = UIWebView()
-        let userAgent = webView.stringByEvaluatingJavaScriptFromString("navigator.userAgent")!
-
-        // Extract the WebKit version and use it as the Safari version.
-        let webKitVersionRegex = NSRegularExpression(pattern: "AppleWebKit/([^ ]+) ", options: nil, error: nil)!
-        let match = webKitVersionRegex.firstMatchInString(userAgent, options: nil, range: NSRange(location: 0, length: count(userAgent)))
-        if match == nil {
-            println("Error: Unable to determine WebKit version")
-            return
-        }
-        let webKitVersion = (userAgent as NSString).substringWithRange(match!.rangeAtIndex(1))
+        let currentiOSVersion = floor(NSFoundationVersionNumber)
+        let lastiOSVersion: Double? = NSUserDefaults.standardUserDefaults().doubleForKey("LastNSFoundationVersionNumber")
+        var firefoxUA = NSUserDefaults.standardUserDefaults().stringForKey("UserAgent")
```

Comment:
But, I don't really want to have to duplicate this line
'SDWebImageDownloader.sharedDownloader().setValue(firefoxUA, forHTTPHeaderField: "User-Agent")'
which is why I did it in a var that either gets set inside the userAgent block